model/entities/strategies/trader_strategy.py

```diff
@@ -13,10 +13,12 @@
 from cvxpy import Maximize, Minimize, Problem, Variable
 import cvxpy
 
-from model.types import MentoBuckets, MentoExchangeConfig, Pair
+from model.types import MentoBuckets, Pair
+from model.config_types import MentoExchangeConfig
 if TYPE_CHECKING:
     from model.entities.trader import Trader
```

Comment:
I think the if statement is to prevent circular referencing. My guess is `if` is not compiled and therefore the import of `Trader` does not conflict with other imports.  But  I am sure @bowd can help